scripts/create-tags.gmp

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
This script looks unfinished, was it really intended for this pull request?

Reply:
No. It wasn't.